### PR TITLE
Leave bugs open on graphics

### PR DESCRIPTION
**Pull request by staktrace**
We want to treat graphics like the other integration branches and leave bugs open. We'll use bugherder to close them once they merge to m-c.

### CODE REVIEW

File: pulsebot.cfg.in [hunk 1]
```diff
@@ -34,4 +34,4 @@ server = https://api.pub.build.mozilla.org/treestatus/trees
 server = https://bugzilla.mozilla.org/
 api_key = @bugzilla.api_key@
 pulse = integration/b2g-inbound,integration/fx-team,integration/mozilla-inbound,integration/autoland,mozilla-central,hgcustom/version-control-tools,mozilla-build,webtools/reviewboard,projects/graphics,automation/conduit
-leave_open = integration/*,mozilla-central
+leave_open = integration/*,mozilla-central,projects/graphics
```
Comment (glandium) [inline pulsebot.cfg.in]: Would be better to go with projects/*.

**Comment by staktrace** [comment]
@glandium ping (I updated the patch as you requested ^)

**Comment by staktrace** [comment]
Updated commit message too

**Comment by glandium** [comment]
This should be live. Ping me if pushing to projects/graphics still closes bugs.

**Pull request merged**